linera-client/src/chain_listener.rs

```diff
@@ -598,7 +598,7 @@ impl<C: ClientContext + 'static> ChainListener<C> {
             Ok((certs, Some(new_timeout))) => {
                 info!(
                     "{} blocks created on chain {chain_id}. Will try processing the inbox later \
-                    based on the round timeout: {new_timeout:?}",
+                    based on the round timeout: {new_timeout}",
```

Comment:
While we're at it: can you rewrite this log to:
```rust
info!(
  ?chain_id,
  block_count=?certs.len(),
  ?new_timeout,
  "new blocks created. delay processing inbox to later round timeout"
);
```
?

Reply:
That's fine. I just would like for us to move ALL logs to this approach. When we start grepping through logs it will be easier of we can look for the whole string (rn we can't b/c of the interpolation).